content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,135 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
+                        [-y <field>] [--no-header] [--title <text>]
+                        [--x-label <text>] [--y-label <text>]
+                        [--unset [<property> [<property> ...]]]
+                        target
+
+positional arguments:
+  target                Metric file to set properties to
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_display properties_ (such as `y-label`, `template`, `title`, etc.) each time
+plots are generated with `dvc plot show` or `dvc plot diff`. This command sets
+(or unsets) default display properties for a specific metrics file.
+
+The path to the metrics file `target` is required. It must be listed in a
+[`dvc.yaml`](/doc/user-guide/dvc-file-format) file (see the `--plots` option of
+`dvc run`). `dvc plots modify` adds the display properties to `dvc.yaml`.
+
+The available display properties are expressed as the [options](#options) of
```

Comment:
omit `available` here ... `Property names are passed as option to this command ...`

Reply:
Done in 1970fc3a